src/components/Kv/KvMessage.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+	<div class="kv-message message-text text-center message-text-confirmation" v-if="safeMessage">
+		<div class="kv-message-row">
+			<img class="kv-message-image" src="~@/assets/images/beta-icon.svg">
+			<div class="kv-message-message">
+				<p class="message" v-html="safeMessage"></p>
```

Comment:
Please make this a slot instead of using v-html

Reply:
Updated :+1: